tests/phpunit/tests/image/editor.php

```diff
@@ -627,6 +627,47 @@ public function it_should_remove_the_webp_version_of_the_image_if_the_image_is_f
 		);
 	}
 
+	/**
+	 * Test removing the backup sources files when image is removed after edited.
+	 *
+	 * @ticket 55443
+	 */
+	public function it_should_remove_the_backup_sizes_and_sources_if_the_attachment_is_deleted_after_edit() {
+		require_once ABSPATH . 'wp-admin/includes/image-edit.php';
+
+		$attachment_id = $this->factory->attachment->create_upload_object(
+			DIR_TESTDATA . '/tests/testdata/modules/images/leafs.jpg'
+		);
```

Comment:
It's a wrong image path that needs to be updated.

Reply:
Fixed the image path there. good catch 👍

Reply:
👍